### PR TITLE
DLS-6373 Upgrade to Scala 2.13

### PR DESCRIPTION
**Pull request by LAJW**

### CODE REVIEW

**Comment by johno1985** [comment]
Can we address some of the PR bot comments?

**Comment by LAJW** [comment]
I've addressed everything except updating the SBT version - this requires non-trivial dependency resolution.

**Comment by johno1985** [comment]
Can these two be easily adressed?

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -7,10 +7,11 @@ object AppDependencies {
   val hmrcMongo = "uk.gov.hmrc.mongo"
   val akka = "com.typesafe.akka"
   val mongoVersion = "0.73.0"
+  val bootstrapVersion = "7.8.0"
```
Comment (johno1985) [inline project/AppDependencies.scala:10]: we should probably try increasing this to at least 7.11.0 to get clear of the http-verbs header bug

**Review by johno1985 — APPROVED** [review]

**Pull request merged**